src/styles/globals.css

```
@@ -360,3 +370,25 @@
 [data-callout-type="danger"] [data-callout-title] {
   color: hsl(350 89% 60% / 0.9);
 }
+
+/* Page-load animation */
+@keyframes fadeInUp {
```

Stylelint (17.3.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)